lib/rules/declaration-property-value-no-unknown/index.mjs

```diff
@@ -74,6 +74,7 @@ const rule = (primary, secondaryOptions) => {
 			'text-box-trim': 'none | trim-start | trim-end | trim-both',
 			'view-timeline':
 				"[ <'view-timeline-name'> [ <'view-timeline-axis'> || <'view-timeline-inset'> ]? ]#",
+			'font-size': '| math',
```

Comment:
Let's match ours.
i.e.
https://github.com/stylelint/stylelint/blob/6137c5249f317747da1e8fceb82c9f3ad3f791d4/lib/reference/keywords.mjs#L122-L136

```suggestion
			'font-size': '| math | -konq-xxx-large | -webkit-xxx-large',
```

Reply:
Let's do this in another PR/issue as it changes the scope of this one.